src/agent/debuglet.ts

```diff
@@ -403,6 +407,12 @@ export class Debuglet extends EventEmitter {
       // This is ignorable.
     }
 
+    if (utils.satisfies(process.version, '>=10 <10.15.3 || >=11 <11.7')) {
```

Comment:
This looks like it can be consolidated into a single if statement.

Reply:
Done